content/guides/kubernetes/storage-integration.md

```diff
@@ -39,7 +39,7 @@ interface](https://github.com/container-storage-interface/spec) (CSI) to provide
 storage functionality to containers. Storage is implemented in CSI plugins. This
 interface / plugin model enables Kubernetes to support many storage options
 implemented via plugins (or drivers) such as
-[vSphere](https://github.com/kubernetes-sigs/vsphere-csi-driver),
+[vSphere](https://github.com/kubernetes-sigs/vsphere-csi-driver), [DellEMC](https://dell.github.io/storage-plugin-docs/docs/installation/helm/)
```

Comment:
Replace url with : https://dell.github.io/storage-plugin-docs/docs/dell-csi-driver/

Reply:
The plan is to bring a single URL where instantly the customer can see all our CSI drivers and jump on the corresponding installation guide for their array. For this reason, original link is better https://dell.github.io/storage-plugin-docs/docs/installation/helm/